### ISSUE TITLE
Add an example to show how to read bytes from wasm memory 

### ISSUE THREAD

**Issue by AntsyLich** [CharlieTap/chasm#34]

**Comment by AntsyLich** [comment]
Unrelated but does K/Wasm support exporting memory? There's seems to be a very recent issue which mentions importing/exporting memory (https://youtrack.jetbrains.com/issue/KT-73234)

**Comment by CharlieTap** [comment]
Yes, it exports the memory (theres only one at present) with the name "memory". You can check for yourself by turning your .wasm file into a .wat file using a cli tool like [wasm-tools](https://github.com/bytecodealliance/wasm-tools):

```shell
wasm-tools parse ./module.wasm
```

You should see an export entry like this:

```wat
(export "memory" (memory 0))
```

**Comment by mipastgt** [comment]
I'd very much appreciate it if you could provide a complete but simple example of how to call a wasm function which takes a ByteArray as input and returns another ByteArray as output. That would open the door for testing chasm with non-trivial code examples.

**Comment by CharlieTap** [comment]
Documentation for working with strings which covers this scenario was added in [this commit](https://github.com/CharlieTap/chasm/commit/8ef18e56f2e5cfa4742a6ff0f718170b1511f2d0)
